shared/engine/session.js

```diff
@@ -158,7 +161,8 @@ class Session {
   }
 
   // We have an incoming call tied to a sessionID, called only by engine
-  incomingCall(method: MethodKey, param: Object, response: ?Object): boolean {
+  incomingCall(method: MethodKey, param: Object, _response: ?Object): boolean {
+    let response = _response
```

Comment:
why have both `_response` and `response`?

Reply:
oh thats leftover, i was nulling it out under some circumstances and we have flow assume parameters are const so i had to have another var, but thats been removed so i'll just remove it, tnx!